packages/x-date-pickers-pro/src/DateRangePicker/DateRangePicker.tsx

```diff
@@ -144,6 +144,12 @@ DateRangePicker.propTypes = {
    * Defaults to localized format based on the used `views`.
    */
   format: PropTypes.string,
+  /**
+   * Density of the format when rendered in the input.
+   * When "spacious", a space will be added before and after each `/`, `-` and `.` character.
+   * @default "normal"
```

Comment:
I think your js doc is outdated 😇

Reply:
Indeed, great observation, it should be `dense` 🙈

Reply:
:angel: